src/FSharpPlus/Control/Collection.fs

```diff
@@ -28,6 +28,7 @@ type OfSeq =
     static member        OfSeq ((x: seq<KeyValuePair<'k,'v>>, _: IDictionary<'k,'v>              ), _: Default3) = x |> Seq.map (|KeyValue|) |> dict
     static member        OfSeq ((x: seq<'k*'v>              , _: IDictionary                     ), _: Default3) = let d = Hashtable () in x |> Seq.iter d.Add; d :> IDictionary
     static member        OfSeq ((x: seq<KeyValuePair<'k,'v>>, _: IDictionary                     ), _: Default3) = let d = Hashtable () in x |> Seq.iter (function (KeyValue x) -> d.Add x); d :> IDictionary
+    static member        OfSeq ((x: seq<'t>                 , _: IReadOnlyCollection<'t>         ), _: Default3) = IReadOnlyCollection.ofSeq x
```

Comment:
Perhaps we should add a test of this overload? 
This seems to be the place for it:
https://github.com/fsprojects/FSharpPlus/blob/61e0e1cd3688fd94a742077c3cdb5a01d4f2d7bb/tests/FSharpPlus.Tests/General.fs#L536-L537

Reply:
Done